superset/utils/dashboard_import_export.py

```diff
@@ -31,27 +30,29 @@
 logger = logging.getLogger(__name__)
 
 
-def decode_dashboards(o: Dict[str, Any]) -> Any:
+def decode_dashboards(  # pylint: disable=too-many-return-statements
+    o: Dict[str, Any]
+) -> Any:
     """
     Function to be passed into json.loads obj_hook parameter
     Recreates the dashboard object from a json representation.
     """
-    import superset.models.core as models
+    import superset.models.core as models  # pylint: disable=unused-import
```

Comment:
I don't see anything below that's referencing this in any way, i.e. I believe the linter is correct in assuming this can be removed.